dynawo/sources/Models/Modelica/Dynawo/Electrical/Wind/IEC/WPP/WPP4ACurrentSource2015.mo

```diff
@@ -14,12 +14,17 @@ within Dynawo.Electrical.Wind.IEC.WPP;
 
 model WPP4ACurrentSource2015 "Wind Power Plant Type 4A model from IEC 61400-27-1:2015 standard : WT4A, communication modules"
   extends Dynawo.Electrical.Wind.IEC.BaseClasses.BaseWPP;
+  extends Dynawo.Electrical.Controls.IEC.IEC61400.Parameters.QControlParameters2015;
 
   //Uf measurement parameters
   parameter Types.AngularVelocityPu DfMaxPu "Maximum frequency ramp rate in pu/s (base omegaNom)" annotation(
     Dialog(tab = "UfMeasurement"));
+  parameter Boolean Mzc "Zero crossing measurement mode (true = 1 if the wind turbine protection system uses zero crossings to detect the frequency - otherwise false = 0)" annotation(
+    Dialog(tab = "UfMeasurement"));
   parameter Types.Time tfFilt "Filter time constant for frequency measurement in s" annotation(
     Dialog(tab = "UfMeasurement"));
+  parameter Types.Time tphiFilt "Filter time constant for voltage angle measurement in s" annotation(
```

Comment:
could you update the Dynawo.Examples.Wind.IEC.Neplan.WPP4ACurrentSource2015 example, by putting values for Mzc and tphiFilt ?

Reply:
I think this is already done for all the concerned examples